### PR TITLE
Feature/isdb 403

### PR DESCRIPTION
**Pull request by RonanCog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by parvezhmcts — APPROVED** [review]
ok

**Review by adetolacog — COMMENTED** [review]
Reviewed

File: src/test/java/uk/gov/hmcts/reform/bulkscan/config/BulkScanFormValidationConfigManagerTest.java [hunk 1]
```diff
@@ -59,19 +59,17 @@ void testConfigA1ShouldNotBeNull() {
...
-        assertEquals(
-                "applicant_ref", getValidationConfigByFormType(FormType.A1).mandatoryFields.get(2));
         assertEquals(
                 "applicant_postcode",
-                getValidationConfigByFormType(FormType.A1).mandatoryFields.get(3));
+                getValidationConfigByFormType(FormType.A1).mandatoryFields.get(2));
         assertEquals(
-                "applicant_telephone_no",
-                getValidationConfigByFormType(FormType.A1).mandatoryFields.get(4));
+                "applicant_telephone_number",
```
Comment (adetolacog) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/config/BulkScanFormValidationConfigManagerTest.java:71]: Can you create constants for field names?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/controllers/BulkScanA1EndpointTest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    public void shouldValidateA1ErrorBulkScanRequest() throws Exception {
+        String bulkScanValidationRequest = readFileFrom(A1_VALIDATION_ERROR_INPUT_PATH);
+
+        String bulkScanValidationResponse = readFileFrom(A1_VALIDATION_ERROR_OUTPUT_PATH);
+
+        Response response =
+                request.header(AUTH_HEADER, s2sClient.serviceAuthTokenGenerator())
+                        .body(bulkScanValidationRequest)
+                        .when()
+                        .contentType("application/json")
```
Comment (adetolacog) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/controllers/BulkScanA1EndpointTest.java:85]: Should this be JSON_CONTENT_TYPE?

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Pull request merged**